ProductList/product.css

```diff
@@ -0,0 +1,100 @@
+@import url("https://infusify.netlify.app/styles/Infusify.css");
+
+.flex-container{
+    display:flex;
+}
+.flex-rows{
+    flex-direction: row;
+}
+
+.flex-columns{
+    flex-direction: column;
+}
+
+.nav-vertical{
+    grid-area: aside;
+    background-color: var(--container-border-white);
+    /* height: calc(100vh - 9%); */
```

Comment:
Remove unwanted comment

Reply:
ok will do, thank you for pointing it out